.github/pull_request_template.md

```diff
@@ -14,16 +15,28 @@
 - [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
 - [ ] Proper commit message prefix on all commits
 - [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
+- Metrics checks:
+  - [ ] The metrics are still exposed and present in Grafana after the change
+  - [ ] The metrics names didn't change (Grafana dashboards and Prometheus alerts are not affected)
+  - [ ] The metrics names did change (Grafana dashboards and Prometheus alerts were fixed)
+- Logs checks:
+  - [ ] The pods logs do not show any errors
+- Network Policy checks:
+  - [ ] The `NetworkPolicy Dashboard` doesn't show any dropped packages
+- Gatekeeper PSPs checks:
+  - [ ] Gatekeeper PSPs do not block the pods from starting after the change
+- Falco checks:
+  - [ ] No Falco alerts are generated by the change
 - Is this changeset backwards compatible for existing clusters? Applying:
-  - [ ] is completely transparent, will not impact the workload in any way.
-  - [ ] requires running a migration script.
-  - [ ] will create noticeable cluster degradation.
+  - [ ] Is completely transparent, will not impact the workload in any way.
+  - [ ] Requires running a migration script.
+  - [ ] Will create noticeable cluster degradation.
         E.g. logs or metrics are not being collected or Kubernetes API server
         will not be responding while upgrading.
-  - [ ] requires draining and/or replacing nodes.
-  - [ ] will change any APIs.
+  - [ ] Requires draining and/or replacing nodes.
+  - [ ] Will change any APIs.
         E.g. removes or changes any CK8S config options or Kubernetes APIs.
-  - [ ] will break the cluster.
+  - [ ] Will break the cluster.
         I.e. full cluster migration is required.
```

Comment:
Can we drop this one, I think it's a relic from when the kubernetes installer was co-located with apps 👴🏻 
```suggestion
```